packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -716,10 +725,8 @@ class SetAccessibilityFocusExample extends React.Component<{}> {
 
     const onClose = () => {
       if (myRef && myRef.current) {
-        AccessibilityInfo.sendAccessiblityEvent_unstable(
```

Comment:
This new method does the same thing as setAccessibilityFocus does, with the exception that this unstable method is fabric-friendly! Did this method not work for you? We don't want to change it back if we can avoid it!

Reply:
@kacieb Yes, I experienced this issue with the current master branch 

<image src="https://user-images.githubusercontent.com/24992535/111324982-8e2f0180-866b-11eb-9c9c-357326258d79.png" width="300" />

Do you want me to keep this change, reverse it or to investigate the reason?
I can not find `sendAccessiblityEvent_unstable` in our codebase. Thanks a lot :pray: :peace_symbol:

Reply:
Ah got it, maybe this only works internally currently. Since this is an RNTester example, it should be ok to keep this change for now, and we can resolve it internally later.

cc @JoshuaGross and @mdvacca for awareness!

Reply:
Thanks @kacieb I reverted this change with commit https://github.com/facebook/react-native/pull/31144/commits/8e76ab3e97597770c378df3a7ec348ee45c35e02 :pray: :peace_symbol:

Reply:
There's a missing "i", it should be : "sendAccessibilityEvent_unstable" instead of "sendAccessiblityEvent_unstable"